test/test/spawn-bail.tap

```diff
@@ -15,8 +15,7 @@ TAP version 13
               ---
               {"found":1,"wanted":"1","compare":"===","at":{"file":"test/test/spawn.js","line":22,"column":10,"function":"foo"},"source":"tt.equal(1, '1', 'nested failure')\n"}
               ...
-            Bail out! # nested failure
-        Bail out! # nested failure
+
```

Comment:
I see a blank line pushed if the child closes before it is `_ended`, and then `bailOnFail` prints `\nBail out!...\n`.. so maybe combination of those.